app/react/kubernetes/components/EventsDatatable/ResourceEventsDatatable.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCurrentStateAndParams } from '@uirouter/react';
+
+import { useKubeStore } from '@/react/kubernetes/datatables/default-kube-datatable-store';
+import { useEvents } from '@/react/kubernetes/queries/useEvents';
+import { EventsDatatable } from '@/react/kubernetes/components/EventsDatatable';
+
+type Props = {
+  storageKey: string;
+  resourceId?: string; // if undefined, all resources for the namespace (or cluster are returned)
```

Comment:
```suggestion
  /** if undefined, all resources for the namespace (or cluster are returned) */
  resourceId?: string; 
```

this will show the comment when hovering the prop when using the component

Reply:
updated